docs/content/foundations/color.mdx

```diff
@@ -4,9 +4,1354 @@ page_title: Color
+        - class_name: 'text-primary'
+          hex_value: '#007bbd'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-primary-vivid'
+          hex_value: '#067cbc'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-primary-dark'
+          hex_value: '#004971'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-primary-darker'
+          hex_value: '#00314b'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-secondary-lighter'
+          hex_value: '#beebee'
+          additional_example_classes: 'radius-md padding-05 bg-gray-90'
+        - class_name: 'text-secondary-light'
+          hex_value: '#4bbfc6'
+          additional_example_classes: 'radius-md padding-05 bg-gray-90'
+        - class_name: 'text-secondary'
+          hex_value: '#298085'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-secondary-vivid'
+          hex_value: '#338084'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-secondary-dark'
+          hex_value: '#1e4c4f'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-secondary-darker'
+          hex_value: '#17373a'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-accent-cool-lighter'
+          hex_value: '#d7e5f4'
+          additional_example_classes: 'radius-md padding-05 bg-gray-90'
+        - class_name: 'text-accent-cool-light'
+          hex_value: '#92a9c8'
+          additional_example_classes: 'radius-md padding-05 bg-gray-90'
+        - class_name: 'text-accent-cool'
+          hex_value: '#5478ab'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-accent-cool-dark'
+          hex_value: '#284976'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-accent-cool-darker'
+          hex_value: '#06162d'
+          additional_example_classes: 'radius-md padding-05'
+        - class_name: 'text-accent-warm-lighter'
+          hex_value: '#fdf2bf'
+          additional_example_classes: 'radius-md padding-05 bg-gray-90'
+        - class_name: 'text-accent-warm-light'
+          hex_value: '#fee685'
+          additional_example_classes: 'radius-md padding-05 bg-gray-90'
+        - class_name: 'text-accent-warm'
+          hex_value: '#face00'
+          additional_example_classes: 'radius-md padding-05 bg-gray-90'
+        - class_name: 'text-accent-warm-dark'
+          hex_value: '#ddaa01'
+          additional_example_classes: 'radius-md padding-05 bg-gray-90'
+        - class_name: 'text-accent-warm-darker'
+          hex_value: '#b38c00'
+          additional_example_classes: 'radius-md padding-05 bg-gray-90'
+        - sub_header: Global Pallette
+        - class_name: 'text-transparent'
```

Comment:
Let's hide text transparent. We don't want people to use it. Ever.

Reply:
This was not done in the token, now it has been.